nemo_curator/core/client.py

```diff
@@ -120,13 +121,18 @@ def start(self) -> None:
                 self.num_cpus,
                 self.enable_object_spilling,
             )
+            # Set environment variable for RAY_ADDRESS
+            ip_address = socket.gethostbyname(socket.gethostname())
```

Comment:
nit: rather than also do this inside `init_cluster` what if we pass the ip_address to the `init_cluster` method that it can re-use